Update guide for next version of Leap Admin (#449)
dwavesystems/dwave-ocean-sdk: docs/leap_sapi/admin.rst

Reviewer: Should the Leap Admin terminology terms be added to [our terminology section](https://docs.dwavequantum.com/en/latest/concepts/index.html)?

An advantage of doing that is that on the first use of a term in a section you could make a definition pop up 

Organization Page
When you open an :term:`organization` ...

Author: Leap Admin terms aren't really used anywhere else.

Reviewer: or organization administrator. 
Do we need to repeat the definition here ("A project administrator for a project is a member who")?

If we add the terminology to the [Concepts](https://docs.dwavequantum.com/en/latest/concepts/index.html) page, users can just hover over the word and a definition will pop up.

Reviewer: Missed comment

Reviewer: Still missed

Reviewer: Is this part helpful? The rest is.

Reviewer: Is this not reversing end and start dates? I mean should this not be like this:

A gap occurs between record 1 and the set of records 2 because the **end** date in record 1 is not contiguous with the closest **start** date in the set of records 2

Author: Hmm... the records are ordered by start date with the one farthest in the future at the top. I guess this is confusing. I can reverse the order.

Reviewer: See previous comment on start and end dates related to arrow

Reviewer: It seems there is also a separate function for inviting a project admin so why is this called out versus inviting a member and assigning the Org Admin role of the next bullet, but not there?

Author: Not user I understood this, but I removed the org-admin-specific points and added a general statement about the hierarchy of the roles.

Reviewer: "Displays the resulting sample set"

What about a problem submitted to the NL solver?

Reviewer: "described "

This reads as saying that solver times are described as "Solver category: QPU, hybrid, or custom", etc

I think you mean they times are broken down in such way?

Reviewer: Let's add a sentence to give some inkling of what this means. Something like "administer an organization and its associated projects"

Reviewer: Let's add a sentence to give some inkling of what this means

Reviewer: Consider changing Terminology to "Terminology and Conventions" and moving this topic's single sentence into that section

Reviewer: It looks to me that only an org admin can view this info and that is not at all clear in this section, which project admins will read too.

Reviewer: The first use of the term "record" in this guide is in Managing Organizations, so a project admin will come to this section and see this first use of record and not know what it means. We should either make it a link to Solver-Access Time Records or make it a term in the terminology or both

Reviewer: "Org Admin role" three instances in this paragraph and nowhere else: Should be "Organization Admin role"

<img width="692" height="190" alt="Image" src="https://github.com/user-attachments/assets/cd1af709-df35-4a8d-a8b2-9b53727167a0" />

*   :ref:`admin_overview` provides an overview of Leap administration, including
    terminology.

> *   :ref:`admin_quick_start` shows how to easily invite people to a

Reviewer: How helpful are lines 15 to 25 given the right-side navigation and that most descriptions don't do much more than spell out what is said in the title?

Consider replacing with the following moved up from Overview, maybe somewhat rephrased ("Access to Leap Admin is provided to users designated as a project or ..."):

If you have access to Leap Admin, you have been designated as a project or organization administrator with privileges to manage one or more projects or organizations that have been created by D-Wave in accordance with your customer contract.

*   Generate solver usage reports.

For information about |dwave_short| quantum computers, problems, and solvers,
see the :ref:`opt_index_get_started` or :ref:`qpu_index_get_started` topics.

To get started quickly, go to the :ref:`admin_quick_start` topic.

Searching Globally
------------------

Via the Global Search field as shown in :numref:`Figure %s <AdminGlobalSearch>`,

> you can search for projects and members across all organizations and projects.

Reviewer 2: Admins cannot search across Orgs using Global search, it is scoped to the selected Org.

*   Generate solver usage reports.

For information about |dwave_short| quantum computers, problems, and solvers,
see the :ref:`opt_index_get_started` or :ref:`qpu_index_get_started` topics.

To get started quickly, go to the :ref:`admin_quick_start` topic.

Searching Globally
------------------

Via the Global Search field as shown in :numref:`Figure %s <AdminGlobalSearch>`,
you can search for projects and members across all projects in the current
organization. The text matches any part of a project, or member name (including
an email address).

.. figure:: ../_images/leap_admin_global_search.png
    :name: AdminGlobalSearch
    :alt: Global Search

    Global Search

.. admonition:: Some Limitations
    :name: global_search_limits

    *   Searching is case-insensitive.

    *   For project names, the Latin-1 character set and the Ä, ä, Ö, ö, ẞ, Ü, ü
        characters are supported; however, diacritical marks are ignored.

>     *   For member names, only the Latin-1 character set is supported.

Reviewer: ```suggestion
    *   Searching is case-insensitive.
     
    * The Latin-1 character set is supported.

    *   For project names, the Ä, ä, Ö, ö, ẞ, Ü, ü characters are also supported;
         however, diacritical marks are ignored.
```

divided by what's in common and what is different

Author: GitHub doesn't allow commits on changes to deleted lines; so I manually implemented this change.

*   Generate solver usage reports.

For information about |dwave_short| quantum computers, problems, and solvers,
see the :ref:`opt_index_get_started` or :ref:`qpu_index_get_started` topics.

To get started quickly, go to the :ref:`admin_quick_start` topic.

Administration Tasks
--------------------

> *   :ref:`Manage organizations <admin_managing_organizations>`

Reviewer: I found this a bit confusing because of the mismatch between the bullet titles (e.g., "Manage organizations" and "Add and manage project members") versus sidebar headings (respectively "Managing Organizations" and "Managing Project Members"). It wasn't clear that the links were to these sections.

<img width="1014" height="521" alt="Image" src="https://github.com/user-attachments/assets/8658f4bf-1b6a-411b-968c-b0a3c465dcba" />

>     You can view information about the organization, such as its name and

Reviewer: Instead of "You" I suggest "Organization administrators" because a project admin reading this cannot do these things.

    <admin_problem_status>`

    You can view and filter a list of problem submissions, access the details
    of any problem, and cancel pending submissions.

*   :ref:`Generate and export usage reports <admin_reports>`

    You can generate usage reports and export them to `XLSX` and `CSV` files.

Searching Globally
------------------

Via the Global Search field as shown in :numref:`Figure %s <AdminGlobalSearch>`,
you can search for organizations, projects, and members across all organizations
and projects. The text matches any part of an organization, project, or member
name (including an email address).

.. figure:: ../_images/leap_admin_global_search.png
    :name: AdminGlobalSearch
    :alt: Global Search

    Global Search

.. admonition:: Some Limitations
    :name: global_search_limits

    *   The search is case-insensitive.

    *   Only the Latin-1 character set is supported.

    *   The maximum number of characters is 254.

.. tip::
    *   Use the `Up` and `Down` arrow keys to select an item in the results list
        and press `Enter` to go to the selection.

Organization Page
-----------------

> When you open an organization, the organization page is displayed as shown in

Reviewer: A project admin does not see this page. To prevent confusion it might be worth stating at the top something like "This page is accessible to organization administrators only"

:numref:`Figure %s <AdminOrgPage>`.

.. figure:: ../_images/leap_admin_org_one.png
    :name: AdminOrgPage
    :alt: Leap Admin Organization Page

    Leap Admin: Organization Page

Tabs that organize tasks into logical groups are as follows:

*   **Projects**

    Displays summary information for the projects in the organization. You can
    display details about a project by clicking its name.

*   **Memberships**

    Displays summary information for all project members across the organization.

>     You can display details about a member by clicking their member name.

Reviewer: You can also change things 

<img width="324" height="237" alt="Image" src="https://github.com/user-attachments/assets/add16347-d160-4e89-91b5-92bd43cfa91e" />

    status, and summary information about its projects, such as their status
    and number of members.

*   :ref:`Manage projects <admin_managing_projects>`

    You can view project information such as the project members, the solvers
    available in the project, and the project status. For time-based projects,
    you can also set the default limit for all members' individual solver-access
    time.

*   :ref:`Add and manage project members <admin_managing_project_members>`

    You can add members by inviting them to join a project. You can also
    set a per-member limit on solver-access time that overrides the project
    default.

*   :ref:`View and manage problems submitted to solvers in the projects
    <admin_problem_status>`

    You can view and filter a list of problem submissions, access the details
    of any problem, and cancel pending submissions.

*   :ref:`Generate and export usage reports <admin_reports>`

    You can generate usage reports and export them to `XLSX` and `CSV` files.

Organization Page
-----------------

When you open an organization, the organization page is displayed as shown in
:numref:`Figure %s <AdminOrgPage>`.

> .. figure:: ../_images/leap_admin_org_one.png

Reviewer: This shows a page with the title "Projects". Will an organization administrator also see just the projects page you are showing? (note also the left tabs are open by default)

When I open Leap Admin I see:

<img width="522" height="208" alt="Image" src="https://github.com/user-attachments/assets/77cb56e5-8062-4641-b751-0202936c3bb8" />

Author: I'll update the screenshots as we get closer to release.

    :name: AdminOrgPage
    :alt: Leap Admin Organization Page

    Leap Admin: Organization Page

Tabs that organize tasks into logical groups are as follows:

*   **Projects**

    Displays summary information for the projects in the organization. You can
    display details about a project by clicking its name.

*   **Memberships**

    Displays summary information for all project members across the organization.
    You can display details about a member by clicking their member name.

*   **Problem Status**

    Displays a summary of the statuses of all submitted problems. You can

>     display a problem's parameters and its sample set by clicking its problem ID.

Reviewer: "and its sample "

Nonlinear models don't have a sample

    Displays information about the project.

*   **Memberships**

    Displays summary information for the project members. You can display
    details about a member by clicking their member name.

*   **Problem Status**

    Displays a summary of the statuses of all submitted problems. You can

>     display a problem's parameters and its sample set by clicking its problem ID.

Reviewer: "and its sample"

probably a bunch of places

*   **Reporting**

    Enables generating CSV or Excel reports on solver usage statistics.

> Searching Globally

Reviewer: Should this be above Organization Page and Project Page sections (from top downward)? It's short and useful, so might be good to get to it first

------------------

Via the Global Search field as shown in :numref:`Figure %s <AdminGlobalSearch>`,
you can search for organizations, projects, and members across all organizations

> and projects. The text matches any part of an organization, project, or member

Reviewer: "The text matches " --> should this be "The text creates matches ..." or "The text automatically matches"? 
I read it as passively matches, i.e., some existing text is a match to ....

.. admonition:: Some Limitations
    :name: global_search_limits

    *   The search is case-insensitive.

    *   Only the Latin-1 character set is supported.

    *   The maximum number of characters is 254.

.. tip::
    *   Use the `Up` and `Down` arrow keys to select an item in the results list
        and press `Enter` to go to the selection.

Filtering and Sorting Data
--------------------------

Leap Admin's user interface is quite intuitive; you can filter and sort data
using similar actions on the various tabs and sections (e.g., **Projects**,
**Problem Status**, and **Solver Access Time Schedule**) by clicking the icons
in the table headings or above the table. Some icons are hidden until you place
the pointer over data that you want to act on; for example, placing the pointer
over a problem ID displays the copy icon |copy_icon|.

Some actions include the following:

*   Filtering rows by matching data in the rows

*   Pinning a column to the left or right side of the table

*   Setting units for the data (e.g., time in units of seconds,
    minutes, hours, or all of them)

*   Sorting data in columns in ascending or descending order

*   Hiding and showing columns

> Terminology

Reviewer: Would it make sense to put the Terminology section before the Administration Tasks section? All the preceding sections use this terminology.

Reviewer: Going through this a second time I more strongly believe the topology shoudl move up to the start

    Displays a summary of the statuses of all submitted problems. You can
    display a problem's parameters and, if available for the solver, the
    problem's sample set by clicking the problem ID.

*   **Reporting**

    Enables generating CSV or Excel reports on solver usage statistics.

*   **Project Members**

    Displays summary information for the project members. To display more
    information about a specific member, click their member name.

Global Search and Profile Avatar Menu
-------------------------------------

.. figure:: ../_images/leap_admin_org_proj_pages_common.png
    :name: AdminProjectOrgCommonPage
    :alt: Global Search and Profile Avatar Menu

    Global Search and Profile Avatar Menu

You can perform the following tasks via global search and the profile avatar
menu on the organization and project pages:

*   Global search field

    Search for a project by name and for a member in any of your projects.

*   Profile avatar menu

    From your profile avatar, select the appropriate menu item to go to the
    **dashboard**, this documentation, open a project, or log out of Leap
    Admin.

Administration Tasks

> --------------------

Reviewer: So for a project admin, the pages as reachable from the left are Projects (page described above) but also Memberships and Problem Statuses, which are not. We should say something to explain why.

<img width="539" height="365" alt="Image" src="https://github.com/user-attachments/assets/8cddb02a-d065-42a4-a6a6-189ef4e843b1" />

Author: Removed this section.

> **Problem Status**, and **Solver Access Time Schedule**) by clicking the icons

Reviewer: I found this unhelpful because Projects and Problem Status are in front of me in the lefthand nav bar but Solver Access Time Schedule is rarely in front of me. 
I'd drop the entire parenthesis, giving examples here won't help the reader.

Author: Not all functionality is included in the list.

> Some actions include the following:

Reviewer: Not really what you wanted to say. Maybe "Available actions include the following:"

Author: Not all functionality is included in the list.

.. _admin_def_solver_access_limit:

> *   **Solver-access limit**

Reviewer 2: I think there were no changes to this section but the second half of this statement is not correct in many cases "This maximum cannot be greater than the project’s total amount of solver-access time, **which is unlimited for most seat-based projects.**". Monthly SAT is not unlimited in most seat-based projects.

institution in the Leap service. An organization contains one or more projects.

.. note::
    *   Only organization administrators can view organizations.

    *   To create an organization, contact |support_email|_.

When you open an organization, the organization page is displayed as shown in
:numref:`Figure %s <AdminOrgPage>`, where tabs organize tasks into logical groups.
If you have more than one organization, you can select one via a dropdown list.

.. figure:: ../_images/leap_admin_org.png
    :name: AdminOrgPage
    :alt: Leap Admin Organization Page

    Leap Admin: Organization Page

Organization Administrators
---------------------------

:ref:`Organization administrators <admin_def_organization_administrator>` can
invite people to the Leap service in any :ref:`role <admin_def_role>` as well as
administer many aspects of the projects contained in their organization,
including performing all tasks that a project administrator can.

Viewing Summary Information for Projects
----------------------------------------

The organization's **Projects** tab displays summary information for all
projects in an organization.

> You may find the following field descriptions helpful:

Reviewer 2: I think it might be useful to add that these fields for project summary are customizable. What do you think?

Author: This section is in the summary list of projects on the org page and we can't change any info here. I believe customers can only change the customer reference ID field on the project details page and I noted that in the section.

Reviewer 2: @qwriter as these fields talk about columns on projects page in Organization, I was saying these columns can now be customized.

<img width="749" height="271" alt="Image" src="https://github.com/user-attachments/assets/5c1ff741-5df0-4e39-a82e-c3e32c51095c" />

Author: Thanks! I added customizing the arrangement of columns to the section that describes customizing how data is displayed.

    appropriate selections.

    You may find the following field descriptions helpful:

    *   **Role**: A *role* gives a member the privileges to perform specific
        actions in a project. For more information, see :ref:`admin_user_roles`.

    *   **Seat Type**: For more information, see the following sections:

        *   :ref:`admin_seat_types`

        *   :ref:`admin_inviting_users_join`

    .. tip::
    
        *   As a best practice, invite additional project and organization
            administrators as backups.

        *   If the desired email address is not allowed as specified
            (individually or by pattern) in the **Allowed Email Settings**
            section on the **Overview** page, contact |support_email|_.

.. _admin_managing_organizations:

Managing Organizations
======================

A customer contract defines an organization, which represents a company or
institution in the Leap service. An organization contains one or more projects.

.. note::
    *   Only organization administrators can view organizations.

    *   To create an organization, contact |support_email|_.

> When you open an organization, the organization page is displayed as shown in

Reviewer: What does it mean to "open an organization"? I think you mean that you click on the name of the organization in the top of the left-side tab (which I don't know if that has a name)

2.  In the upper right corner of the dashboard, select the Profile Avatar icon >
    **Leap Admin** as shown in :numref:`Figure %s <DashboardLeapAdminLogin>`.

    .. _fig_dashboard_leap_admin_login:

    .. figure:: ../_images/leap_admin_dashboard_login.png
        :name: DashboardLeapAdminLogin
        :alt: Leap Admin Login Menu
        :width: 33 %

        Leap Admin: Login Menu

.. note::
    If you are a project administrator for multiple projects,
    select the Profile Avatar icon > **Leap Admin** > *project_name*.

.. _admin_project_inviting_users_join:

Inviting People to a Project
----------------------------

#.  In the upper-right corner, click |plus_icon| > **Invite User** and make the
    appropriate selections.

    You may find the following field descriptions helpful:

    *   **Role**: A *role* gives a member the privileges to perform specific
        actions in a project. For more information, see :ref:`admin_user_roles`.

    *   **Seat Type**: For more information, see the following sections:

        *   :ref:`admin_seat_types`

        *   :ref:`admin_inviting_users_join`

    .. tip:: 

>         

Reviewer: ```suggestion
    .. tip::
    
```

Trailing spaces

1.  Log in to the Leap service at `<https://cloud.dwavesys.com/leap/login/>`_.

2.  In the upper right corner of the dashboard, select the Profile Avatar icon >
    **Leap Admin** as shown in :numref:`Figure %s <DashboardLeapAdminLogin>`.

    .. _fig_dashboard_leap_admin_login:

    .. figure:: ../_images/leap_admin_dashboard_login.png
        :name: DashboardLeapAdminLogin
        :alt: Leap Admin Login Menu
        :width: 33 %

        Leap Admin: Login Menu

.. note::
    If you are a project administrator for multiple projects,
    select the Profile Avatar icon > **Leap Admin** > *project_name*.

.. _admin_project_inviting_users_join:

Inviting People to a Project
----------------------------

#.  In the upper-right corner, click |plus_icon| > **Invite User** and make the
    appropriate selections.

>     Descriptions of the following selections may be helpful:

Reviewer: I love that we're describing only non-intuitive selection but find the intro sentence a bit off. 
Can we change to something like "These selections include the following:"?

    To create an organization, contact |support_email|_.

> 

Reviewer: This section probably needs a note that only org admins have access to these pages

including specifying a project's allowed email addresses and performing all
tasks that a project administrator can.

Viewing Summary Information for Projects
----------------------------------------

The organization's **Projects** tab displays summary information for all
projects in an organization.

> Descriptions of the following columns may be helpful:

Reviewer: Same as before, something a bit off about this phrasing

*   **Monthly Usage**: Amount of solver-access time that has been consumed in
    the project for the month.

*   **Solver Access Time**: Amount of solver-access time aggregated over all
    active records for the project.

*   **Project Status**: Status of the project. For a description of project
    statuses, see the :ref:`admin_proj_status_viewing` section.

> 

Reviewer: Concurrency also maybe?

Author: Concurrency isn't displayed for customers.

        -   The end date of all solver-access time allocations in all the
            organization's projects has been reached.

.. _admin_managing_projects:

Managing Projects
=================

A customer contract defines each project, including the
:ref:`project type <admin_project_seat_time>` (seat- or time-based), the solvers
that are available, and the allowed email addresses for project members.

In addition to viewing the email domains and subdomains (or individual email
addresses) of the people who can be invited (also known as the "allowed email
list") and project-level information such as the available solvers and the
project's :ref:`seats <admin_def_seat>`, you can change a few project
settings, such as its :ref:`customer reference ID <admin_project_info_viewing>`,
members' limits on solver-access time, and

> :ref:`seat types <admin_seat_changing_user>`.

Reviewer 2: Project admins cannot change number or different type of seat, only invite users to different seat types as they want.

*   Time-based projects do not have seats; instead you allocate an amount of
    solver-access time to each project member.

.. _admin_project_info_viewing:

Viewing Basic Information
-------------------------

Basic information about the project, such as its status is displayed on the

> title banner and on the **Overview** tab.

Reviewer: For someone coming here by clicking Managing Projects (right side nav bar), it might not be clear where this Overview is because if they have a few projects the page might look like this:

<img width="819" height="386" alt="Image" src="https://github.com/user-attachments/assets/6ea93c7e-5a89-47d0-8a49-1bf35b20d2bf" />

Probably worth keeping the "project page's **Overview** tab"

*   Time-based projects do not have seats; instead you allocate an amount of
    solver-access time to each project member.

.. _admin_project_info_viewing:

Viewing Basic Information
-------------------------

Basic information about the project, such as its status is displayed on the
title banner and on the **Overview** tab.

You may find the following field descriptions helpful.

.. tabularcolumns:: |l|L|

.. list-table:: **Current Usage** Section (Time-based Projects Only)
    :widths: 1 3

    *   -   **Monthly Usage**
        -   Percent of solver-access time used in the project for the current
            renewal period.

.. tabularcolumns:: |l|L|

> .. list-table:: **Project Seats** Section

Reviewer 2: Is this supposed to be Project Overview Section instead?

*   Time-based projects do not have seats; instead you allocate an amount of
    solver-access time to each project member.

.. _admin_project_info_viewing:

Viewing Basic Information
-------------------------

Basic information about the project, such as its status is displayed on the

> title banner and on the **Overview** tab; descriptions of the following fields

Reviewer: "descriptions of the following fields" same here

on the **Overview** tab may be helpful:

.. tabularcolumns:: |l|L|

.. list-table:: **Current Usage** Section (Time-based Projects Only)
    :widths: 1 3

    *   -   **Monthly Usage**
        -   Percent of solver-access time used in the project for the current
            renewal period.

.. tabularcolumns:: |l|L|

.. list-table:: **Project Seats** Section
    :widths: 1 3

>     *   -   **Project Code**

Reviewer: For a time-based project, this is in the top of the Overview section 
Also I see this for a seat-based project

        -   Unique identifier for a project. It is used for internal purposes,
            such as a prefix for the project's API token.

>     *   -   **Customer Reference ID**

Reviewer: For a time-based project, this is in the top of the Overview section 
Also I see this for a seat-based project

        -   ID that you provide and which can be used for your specific internal

>             processes.

Reviewer: What about these terms in the top of the overview tab:
"Bundle", "Proportional Allocation", and "Concurrency Limit"?  (are they clear?)

Reviewer 2: For project/Org admins these fields are not visible in overview page but there is a Project Type field that can be added here.

    *   -   **Solver Access Limit**

>         -   |solver_access_limit|

Reviewer: Is this needed here?

Author: I don't think the definition is readily apparent.

    *   -   **Description**
        -   The description for the seat type. For more information, see

>             :ref:`admin_seat_types`.

Reviewer: Seems safe to drop, no?

.. tip:: A project administrator receives a notification when the project's
    solver-access time reaches 80%.

You may find the following field descriptions helpful.

.. tabularcolumns:: |l|L|

.. list-table:: **Solver Access Time Schedule** Section

    *   -   **Monthly Solver Access Time**
        -   A record's amount of solver-access time available for each renewal
            period. Consumption of solver-access time is aggregated across all
            regions.

            **Note:** The maximum number of problems per member that can be in
            a solver queue is 1,000; if this maximum is exceeded, then problems
            are rejected.

    *   -   **Allocation Start Date (UTC)**
        -   The date (inclusive) that access is provided to solvers. This date
            is determined by the customer contract.

    *   -   **Allocation End Date (UTC)**
        -   The date (inclusive) that access to solvers ends. This date is
            determined by the customer contract.

    *   -   **Anniversary Date (UTC)**

>         -   The anniversary date is used to calculate the renewal date.

Reviewer 2: May be we can clarify here that it is used to calculate the monthly solver access time renewal date.

.. tip:: A project administrator receives a notification when the project's
    solver-access time reaches 80%.

You may find the following field descriptions helpful.

.. tabularcolumns:: |l|L|

.. list-table:: **Solver Access Time Schedule** Section

    *   -   **Monthly Solver Access Time**
        -   A record's amount of solver-access time available for each renewal
            period. Consumption of solver-access time is aggregated across all
            regions.

>             **Note:** The maximum number of problems per member that can be in

Reviewer: If it's generally true that the maximum number of problems a user can submit to a solver queue is 1000, seem that [Leap Service page](https://docs.dwavequantum.com/en/latest/leap_sapi/index.html) is the better place for this info.

Author: Created JIRA ticket.

have gaps between them. :numref:`Figure %s <ExampleSolverAccessTimeRecords>`
illustrates the following:

*   A gap occurs between the set of records 1 and record 2 because the latest
    end date in the set of records 1 is not contiguous with the start date in
    record 2.

*   The records in the set of records 1 are contiguous.

*   In the set of records 1, the first and second records overlap.

> .. figure:: ../_images/leap_admin_solver_time_alloc_record.png

Reviewer 2: Customer admin do not see contract ID column that is included in screenshot.

*   The anniversary date is the same as the existing active or scheduled records
    if the new record is contiguous with those existing records.

Viewing Available Solvers
-------------------------

> You view a list of solvers that are available in the project in the **Overview**

Reviewer: Both the Organizations page and the page for a particular project have Overview tabs. So I think you need to specify here.

.. note:: This feature applies to time-based projects only.

By default, any one project member can use up all of a project's monthly
solver-access time\ [#]_ as determined by the sum of the project's active
solver-access time records. If you want to set a lower limit for members,
do the following:

> #.  Go to the project page and deselect **Edit Project** >

Reviewer: "deselect"?

solver-access time. If you want to change this default or set a different limit
for an individual member, do the following:

#.  Edit the project page and deselect the **Set Default Solver Access to Unlimited** 
    option.

#.  In the **Default Solver Access Limit** field, specify the default limit on
    solver-access time for new members who accept the invitation for the project
    after this field has been set. Existing members are not impacted; they retain
    the current limit on their solver-access time.

    You can override this default limit for an individual member; to do so, see
    the :ref:`admin_user_solver_time` section.

.. [#] Solvers have different usage rates. For more information, see the

>     :ref:`qpu_operation_timing` section.

Reviewer: This footnote is now detached from the main text 

<img width="760" height="215" alt="Image" src="https://github.com/user-attachments/assets/fdda5484-b6cb-4e01-a1dd-276969d7cb69" />

Member Roles and Privileges

> ---------------------------

Reviewer: Something to think about: would this page work better if subsection Member Roles and Privileges was moved to under Overview? This informational subsection and the Seat Types would be easier to find under Overview, and then you just link to them in other places. Also, someone reading through for the first time gets those definitions up front.

Procedure
~~~~~~~~~

To invite people to join a project or organization, click |plus_icon| >
**Invite User** and specify the fields.

You may find the following field descriptions helpful.

.. tabularcolumns:: |l|p{9cm}|

> .. list-table:: Invitation Dialog Box

Reviewer: This table is misformatted and does not show up because the last two rows are not indented:

<img width="762" height="281" alt="Image" src="https://github.com/user-attachments/assets/4d30a85f-4bc9-44a5-a289-728f0f7a2c5d" />

Rows Organization to email are missing. 

Making this comment before beginning to rereview so you can fix.

Procedure
~~~~~~~~~

To invite people to join a project or organization, click |plus_icon| >
**Invite User** and specify the fields.

You may find the following field descriptions helpful.

.. tabularcolumns:: |l|p{9cm}|

.. list-table:: Invitation Dialog Box

    *   -   **Seat Type**
        -   An available seat type for the invitees. For more information,
            see the :ref:`admin_seat_types` section.

    *   -   **Email**
        -   Email addresses or member names (if they already exist in the
            Leap service) of the people you want to invite to the project.

            **Tip:**

            *   You can copy and paste multiple email addresses from either
                a single column in an Excel spreadsheet or a semicolon-,
                space-, new line-, or comma-delimited list (for example, a
                CSV file).

            **Note:** If the desired email address is not allowed as specified
            (individually or by pattern) in the **Allowed Email Settings**
            section on the **Overview** page, contact |support_email|_.

    *   -   **Role**

>     

Reviewer: ```suggestion

```

Trailing spaces

        -   One of the project member :ref:`roles <admin_user_roles>` to assign
            to a person: **User**, **Project Admin**, or **Organization Admin**.

>     

Reviewer: ```suggestion

```

Trailing spaces

Sending Reminders
-----------------

On the **Memberships** tab, you can send invitation reminders to one or more
invitees.

*   To send an invitation reminder to an invitee,  in the **Actions** column,
    click |meatball_menu| > **Send Invitation Reminder**.

*   To send invitation reminders to multiple invitees simultaneously, select the

>     invitees to which to send invitation reminders and use the *Manage Selected*

Reviewer: "select the invitees" --> on the Memberships page I don't see an intuitive way to select multiple members. Do you mean here "Filter the invitees"?

Author: There should be a checkbox next to the Users column heading. 
<img width="103" height="166" alt="image" src="https://github.com/user-attachments/assets/748e70e3-260b-44ea-ac1b-38f1030f2d1a" />

    icon |multiselect_menu|.

Resending Invitations
---------------------

On the **Memberships** tab, you can resend invitations that have expired.

*   To resend an invitation for an invitee, click |meatball_menu| >
    **Resend Invitation**.

*   To resend the invitation to multiple invitees simultaneously, select the

>     invitees for which to resend invitations and use the *Manage Selected* icon

Reviewer: "select the invitees" --> on the Memberships page I don't see an intuitive way to select multiple members. Do you mean here "Filter the invitees"?

Author: See this [comment](https://github.com/dwavesystems/dwave-ocean-sdk/pull/449#discussion_r3185142923).

        -   Displays the amount of solver-access time that is allocated to a
            member.

    *   -   **Status**
        -   The status of the member in the project. For more information, see
            :ref:`admin_member_status_viewing`.

To display information about a specific member, click the project's
**Memberships** tab > *member_name*.

.. _admin_seat_changing_user:

Changing a Project Member's Seat Type
-------------------------------------

On the project's **Memberships** tab, click |meatball_menu| for a member and

> select **Change Seat Type**.

Reviewer: Is it clear that this option won't display for a time-based project?

Author: Yes, I think it's self-evident that time-based projects don't have seats. At any rate, in the UI, the user won't see this option for time-based projects.

Modifying the Limit on a Member's Solver-Access Time
----------------------------------------------------

On the **Memberships** tab, you can modify the limit on solver-access time for
one or more members simultaneously.

#.  Do one of the following:

    *   For a member, click |meatball_menu| > **Modify Solver Access Limit**.

    *   For multiple members simultaneously, select the members to modify and

>         use the *Manage Selected* icon |multiselect_menu|.

Reviewer: Same regarding "select the members"

Author: See this [comment](https://github.com/dwavesystems/dwave-ocean-sdk/pull/449#discussion_r3185178158).

    *   **Unlimited Access to Project Solver Time** (time-based projects only):
        Sets the maximum amount of solver-access time to the full monthly
        solver-access time allocated to the project as determined by the sum of
        the project's active solver-access time records.

.. _admin_user_role_change:

Changing a Member's Role
------------------------

On the **Memberships** tab, click |meatball_menu| > **Assign Role** for a member
and select the appropriate role.

The following limitations apply:

*   You can only assign a role at your level or lower; for example, only an
    organization administrator can assign the *Organization Admin* role. 

*   You cannot remove your own role. Instead, ask another organization or
    project administrator to do so.

*   Removing the Organization Admin Role changes the member's role to the
    member's previous role.

Removing Members From a Project
-------------------------------

On the **Memberships** tab, you can remove one or more members simultaneously.

.. note:: You can only remove members whose membership status is *Active*.

*   For a member, click |meatball_menu| > **Remove from project**.

> *   To remove multiple members simultaneously, select the members to remove and

Reviewer: same

Cancelling Problem Submissions
------------------------------

To cancel pending or in-progress problems, select the problems and click the

> **Cancel Problems** **X**.

Reviewer: Should this be an icon instead of an "X"? I read this as saying there's a "Cancel Problems X" button. The red seems worth capturing too.

<img width="137" height="215" alt="Image" src="https://github.com/user-attachments/assets/a86a1985-0e56-42f4-aa72-09c521ac8b4b" />

Also, whey stop and start the bold mid sentence?

.. tip::

    *   To display the most up-to-date status, click *Refresh Table*

>         |refresh_icon|.

Reviewer: This one is broken due to missing image

    *   To copy a problem ID to the clipboard, place the pointer over the
        problem ID and click the copy icon |copy_icon| that is displayed.

    *   To set the solver region (for example, Europe or North America) to

>         display the problems submitted only to that region's solvers, use the

Reviewer: This comes across as saying that by default all regions are shown but for me at least I see only one region (default N. America)

        *Regions* icon |region_icon|.

> Descriptions of the following columns may be helpful:

Reviewer: Still find this phrasing strange

.. tip::
    *   Before cancelling problem submissions, display the most up-to-date
        status by clicking **Refresh Table** |refresh_icon|.

>     *   If you run scripts to continuously submit problems, refresh the table

Reviewer: "continuously" --> "continually" 

Finally got to enjoy this particular nitpick,
